internal/metadata/clustermdloader_test.go

```diff
@@ -49,6 +56,60 @@ func TestUpdatePlan(t *testing.T) {
 		}
 	})
 
+	t.Run("check if all clusters in constant were loaded", func(t *testing.T) {
+
+		type Teststruct struct {
+			//Kind string `yaml:"kind"`
+			Metadata struct {
+				Name string `yaml:"name"`
+			} `yaml:"metadata"`
+		}
+
+		p := plantypes.NewPlan()
+		loader := metadata.ClusterMDLoader{}
+		cmMap := loader.GetClusters(p)
+		//fmt.Println(cmMap)
+		files, err := ioutil.ReadDir("clusters/")
+		if err != nil {
+			fmt.Println("error reading clusters folder")
+		}
+
+		// filter the yaml files
+		re_yaml := regexp.MustCompile(".yaml")
+		re_yml := regexp.MustCompile(".yml")
+
+		var yfiles []string
+		for _, file := range files {
+			if re_yaml.MatchString(file.Name()) || re_yml.MatchString(file.Name()) {
+				yfiles = append(yfiles, file.Name())
+			}
+		}
+
+		// parse yaml files and collect the required info (medata-> names)
+		var metadata_names []string
+		fmt.Println("parsing yaml files:")
+		for _, yfile := range yfiles {
+			filename, _ := filepath.Abs("clusters/" + yfile)
+			//fmt.Println(filename)
+			content, _ := ioutil.ReadFile(filename)
```

Comment:
For reading an yaml you can use the below function

https://github.com/konveyor/move2kube/blob/8afca8785e03fc85a6859b2ce6652916043cd016/internal/common/utils.go#L179